pkgs/development/python-modules/emojis/default.nix

```
@@ -0,0 +1,43 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pythonOlder,
+  pandoc,
+  lib,
+}:
+buildPythonPackage {
```

```suggestion
buildPythonPackage rec {
```
To pass the version along

---

Done!